Added Register and Login screens

nirvan73 opened the pull request:
## 📝 Overview
- This PR fixes or fixes part of https://github.com/bsoc-bitbyte/DigiGate/issues/10#
- This PR does the following: Implements the register and login screen based on the provided Figma design , while keeping the colors , theme , fonts and shapes in mind.

---

## ✅ Checklist
- ✅ My code follows the project's style and structure
- ✅ Matches Figma design specifications
- ✅ Tested locally with no errors

---

## 📸 Proof that changes are correct
- Added screenshots of the register and login page layout in both light and dark mode respectively.

- Also the previews for the composable functions implemented inside the RegisterScreen.kt and LoginScreen.kt file are made inside the file itself.

---

## 📌 PR Pointers
- If you need a review or an answer to a question, and don't have permissions to assign people, leave a comment like the following: "{{Question/comment}} @{{reviewer_username}} PTAL".
- Never force push. If you do, your PR will be closed.

---
## 🗒️ Additional Notes
- Added the required images/icons in the drawable file .
- Also had made some necessary changes in the colors of label, text and placeholder because the current ones were causing problem in visibility in the respective dark and light mode of screens .

![Screenshot 2025-06-13 140744](https://github.com/user-attachments/assets/02de0763-6692-4352-96f9-36c9d0512a1a)
![Screenshot 2025-06-13 140837](https://github.com/user-attachments/assets/ed0d35e4-8902-4056-9abf-9e64f9d3f853)

nirvan73 committed:
Added Register and Login Screens Layout

nirvan73 committed:
Merge remote-tracking branch 'upstream/main' into ui/Register-and-Login-screen-layout

Agrawal-Sujal commented on app/src/main/java/com/tpc/digigate/ui/screens/home/LoginScreen.kt:
```diff
@@ -0,0 +1,205 @@
+    email: String,
+    password: String,
+    onEmailInput: (String) -> Unit,
+    onPasswordInput: (String) -> Unit,
```
```kotlin
email: String,
password: String,
onEmailInput: (String) -> Unit,
onPasswordInput: (String) -> Unit,
 ```

remove all these parameters

Agrawal-Sujal commented on app/src/main/java/com/tpc/digigate/ui/screens/home/LoginScreen.kt:
```diff
@@ -0,0 +1,205 @@
+            .fillMaxSize()
+            .background(MaterialTheme.colorScheme.background)
+            .padding(horizontal = 16.dp, vertical = 24.dp),
+        verticalArrangement = Arrangement.Bottom,
```
`verticalArrangement = Arrangement.Bottom,` remove this line
remove vertical padding

Agrawal-Sujal commented on app/src/main/java/com/tpc/digigate/ui/screens/home/LoginScreen.kt:
```diff
@@ -0,0 +1,205 @@
+        Surface(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(418.dp),
```
remove height

Agrawal-Sujal commented on app/src/main/java/com/tpc/digigate/ui/screens/home/LoginScreen.kt:
```diff
@@ -0,0 +1,205 @@
+            shape = RoundedCornerShape(30.dp)
+        ) {
+            Column(
+                modifier = Modifier.fillMaxSize(),
```
remove fillMaxSize()

Agrawal-Sujal commented on app/src/main/java/com/tpc/digigate/ui/screens/home/LoginScreen.kt:
```diff
@@ -0,0 +1,205 @@
+                            style = MaterialTheme.typography.titleMedium
+                        )
+                    }
+                }
```
Add Spacer here

Agrawal-Sujal replied:
Also make all the changes in Register.kt

Agrawal-Sujal commented on app/src/main/java/com/tpc/digigate/ui/screens/home/LoginScreen.kt:
```diff
@@ -0,0 +1,205 @@
+                    )
+                ) {
+                    Text(
+                        text = "Login",
```
Define String in Strings.xml

Agrawal-Sujal commented on app/src/main/java/com/tpc/digigate/ui/screens/home/LoginScreen.kt:
```diff
@@ -0,0 +1,205 @@
+package com.tpc.digigate.ui.screens.homeScreen
```
Create a new package named authentication and add these two files there.

nirvan73 committed:
Updated the Login and Register screens

Updated as per the asked changes and also added the two files in a new package in screens named authentication

nirvan73 commented:
Updated as per asked changes .

Agrawal-Sujal committed:
Delete .idea/material_theme_project_new.xml

Agrawal-Sujal commented:
LGTM
